applications/MultilevelMonteCarloLaplacianApplication/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+message("**** configuring KratosMyMultilevelMonteCarloLaplacianApplication ****")
+
+include_directories( ${CMAKE_SOURCE_DIR}/kratos )
+
+# generate variables with the sources
+set( KRATOS_MY_MULTILEVEL_MONTE_CARLO_LAPLACIAN_APPLICATION_SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/my_multilevel_monte_carlo_laplacian_application.cpp
```

Comment:
as commented below, i would take out the "my"

also... this is not only a "laplacian application", it implementes the framework for "multilevelmontecarlo" so i would call it 

MultiLevelMontecarlo Application